step_definitions/functions/pageAssertions.ts

```diff
@@ -94,7 +94,6 @@ export async function checkElementEnable(self: World, elementType: string, typeV
 
 export async function isElementDisplayed(self: World, elementType: string, typeValue: string) {
   debugLog(`checking element displayed status ${elementType} ${typeValue}`);
-  await waitForElementToBeLocated(self, elementType, typeValue, 4000);
```

Comment:
if you are asserting something doesn't exist this times out and kills the tests.